examples/classic_controllers/classic_controllers_synch_motor_example.py

```diff
@@ -0,0 +1,69 @@
+from classic_controllers import Controller
+from externally_referenced_state_plot import ExternallyReferencedStatePlot
+import gym_electric_motor as gem
+from gym_electric_motor.visualization import MotorDashboard
+import numpy as np
+
+if __name__ == '__main__':
+
+    """
+        motor type:     'PMSM'      Permanent Magnet Synchronous Motor
+                        'SynRm'     Synchronous Reluctance Motor
+                        
+        control type:   'S'         Speed Control
+                        'T'         Torque Control
+                        'C'         Current Control
+
+        action_type:    'AbcCont'   Continuous Action Space in ABC-Coordinates
+                        'Finite'    Discrete Action Space
+    """
+
+    motor_type = 'PMSM'
+    control_type = 'S'
+    action_type = 'AbcCont'
```

Comment:
The following combinations resulted in an error:

- PMSM + S + Finite  (issue with automated controller staging)
- PMSM + C + Finite (issue with automated controller staging)
- All SynRm options (no registered env. error)

 Please resolve these issues and/or add a comment to this example file which configurations are actually working at the moment

Reply:
The key for the synchronous reluctance motor is SynRM and not SynRm. With this key the environments are executable.#
The staging error appears for the finite control environments of this motor type, too.

Furthermore, the control performance looks not satisfying to me.

There are multiple current limit violations in the current control env. I think I should have a look at the motor parameters here. The current limit is too close to the nominal value of the currents.

Additionaly, there is a steady state error in the torque control application. 

Finally, the reference of the speed control environment is still too dynamic. However, I think the controller could also act a little more aggressive.

Reply:
I have found the bug of the synchronous motors. I will have a look on the torque control of the SynRM.

Reply:
It was just an error in the manual definition of the environment. A psi_p was also passed for the SynRM, which caused the torque control error. The SynRM also had an error with the current limitation of the torque control. This has also been fixed.